src/ReactWrapper.js

```diff
@@ -54,10 +54,12 @@ export default class ReactWrapper {
     }
 
     if (!root) {
+      const ReactWrapperComponent = createWrapperComponent(nodes, options);
       this.component = renderIntoDocument(
         <ReactWrapperComponent
           Component={nodes.type}
           props={nodes.props}
+          context={options.context}
           />
```

Comment:
not due to this diff, but the airbnb styleguide shows this closing wrapper to be back 2 spaces. https://github.com/airbnb/javascript/tree/master/react#alignment